helm-chart/values.yaml

```diff
@@ -46,9 +46,9 @@ securityContext: {}
 pvc:
   storageClassName: standard
   conf:
-    size: 1Gi
+    size: 10Gi
```

Comment:
How will a config file will ever get this big?

Reply:
No, we were talking about the `data` volume. 1Gi for `config` wil be more than enough..

Reply:
My bad, fixed :)